packages/mobile/src/navigator/Navigator.tsx

```diff
@@ -504,7 +503,7 @@ export function MainStackScreen() {
     } else if (!e164Number || !acceptedTerms || pincodeType === PincodeType.Unset) {
       // User didn't go far enough in onboarding, start again from education
       initialRoute = Screens.OnboardingEducationScreen
-    } else if (!redeemComplete && !account) {
+    } else if (!redeemComplete) {
```

Comment:
@jeanregisser similar to what we discussed earlier, I don't think this `account` is necessary. I am removing it because if there is an invite code redemption failure, it often fails after an account has been created but before the invite funds have been transferred. So if a user closes the app after an invite failure, they are skipped ahead to verification instead of staying on the screen and having the opportunity to try again to redeem